### DISCUSSION TITLE
安装后进入系统无显示

### DISCUSSION THREAD

**Discussion by iori2333** [Icarusradio/intel-media-driver-dg1#18] [Q&A]
我在按照[OOT Driver](https://github.com/intel-gpu/intel-gpu-i915-backports)文档安装DKMS驱动后，似乎kernel加载i915驱动后显卡就没有输出了；我手动禁用掉，则可以正常显示。请问有人遇到相同问题了吗？

**Comment by Icarusradio** [comment]
> For Alchemist discrete Graphics cards, support is provided without display. This repo can be used for the features like GPU debug functionality. For normal cases, please use upstream 6.2 or later kernel version.

没有显示输出是正常现象。
